### ISSUE TITLE
In Revit 2023 IfcSpace Qto_SpaceBaesQuantities NetFloorArea and others are not getting exported 

### ISSUE THREAD

**Issue by ArunSanthosh98** [Autodesk/revit-ifc#529]
![IfcSpace_IFC4_Schema_Revit2023_Vs_Revit 2020](https://user-images.githubusercontent.com/110660510/196401960-fcbbecf7-ef5e-46e7-8115-6b7cd6ab5767.PNG)

**Comment by sebortner** [comment]
We have the same problem with IFC4 from Revit 2022. No NetFloorArea is exported.

**Comment by ArunSanthosh98** [comment]
Yes in 2022 any of the Qto_SpaceBaesQuantities roots are not exported

**Comment by ArunSanthosh98** [comment]
![image](https://user-images.githubusercontent.com/110660510/196655810-7c25e0cd-2083-473b-a0ad-c4b6806c27a6.png)

**Comment by ArunSanthosh98** [comment]
@sebortner Do you find any solution for this 
sebortner

**Comment by ArunSanthosh98** [comment]
![image](https://user-images.githubusercontent.com/110660510/197106119-79d85fbb-3a04-4739-ba76-61fe4204971f.png)
covering NetArea is not exporting

**Comment by ArtemHnatko** [comment]
The issue reported — missing _Qto_SpaceBaseQuantities_ properties during IFC4 Reference View export — is **not reproducible** in Revit 2023–2026.Based on testing, the following properties are successfully exported:

- **Revit 2023:** GrossCeilingArea, GrossFloorArea, GrossVolume, Height, NetFloorArea, NetVolume
- **Revit 2024:** GrossCeilingArea, GrossFloorArea, GrossVolume, Height, NetFloorArea
- **Revit 2025:** GrossCeilingArea, GrossFloorArea, GrossVolume, Height, NetFloorArea, NetVolume
- **Revit 2026:** GrossCeilingArea, GrossFloorArea, Height, NetFloorArea, NetVolume

Therefore, the problem originally reported in Revit 2020–2022 has been resolved in current versions.

**Testing environment:**
- Autodesk Revit 2023 (IFC 23.4.1.0)
- Autodesk Revit 2024 (IFC 24.3.20.0)
- Autodesk Revit 2025 (IFC 25.4.0.27)
- Autodesk Revit 2026 (IFC 26.1.0.23)

![Image](https://github.com/user-attachments/assets/411c2b5d-e427-42be-819c-e82cce18af03)
